### ISSUE TITLE
Crash loop

### ISSUE THREAD

**Issue by imReker** [SniperGER/Redstone#34]
Hi, I post the crashreport here because your email server alway reject my mail.

> submissions*festival.ml
> markab.uberspace.de
> Remote Server returned '550 5.0.350 Remote server returned an error -> 553 sorry, this recipient is not in my validrcptto list (#5.7.1)'

[Crash.zip](https://github.com/SniperGER/Redstone/files/1096699/Crash.zip)

**Comment by SniperGER** [comment]
Yeah, notifications have been problematic for a while. I've got a better way for displaying notifications when I started working on volume controls, I just didn't have the time yet to implement this.

I'm sorry this issue is appearing on iOS 9. I should have tested it there before releasing it, but I was in the progress of restoring my development environment and moving to Xcode 9 beta, so I absolutely forgot about iOS 9.

About the mail server issue: I don't know whats happening there. Guess I'll have a look at it.

**Comment by imReker** [comment]
@SniperGER  This project involved with lots of system parts, so I think you should finish features one by one. I don't want this project follow Paragon's death, the author fall into tons of bug and tired to fix them.

**Comment by SniperGER** [comment]
The new notification system doesn't mess with any iOS specific API, so it should fix your system. I did actually test it in the iOS 9 simulator this time, but I couldn't test it on a real device running iOS 9. Could you please check if 17w26a fixes your issues?

**Comment by imReker** [comment]
Crash on weather update.
Crash Source:
https://github.com/SniperGER/Redstone/blob/03ffa23485d7597f752220b0996de9b00dbf8297/Redstone/Tiles/Weather/RSShortLookView.m#L6
`WeatherPreferences userTemperatureUnit: unrecognized selector sent to instance`
